python/JetMaker_cfi.py

```diff
@@ -90,7 +90,7 @@
 
 ### https://twiki.cern.ch/twiki/bin/view/CMS/JetWtagging#Recommendation_for_13_TeV_data_a (W tag)
 defaultCHSHJetSelectionParameters = defaultAK8CHSJetSelectionParameters.clone(
-    jetPtMin            = cms.double(300),
+    jetPtMin            = cms.double(200),
     jetPrunedMassMin    = cms.double(105) ,
```

Comment:
The Higgs pT should be > 300 GeV. Please put it back.
The Z pT can be > 200 GeV